Client/my-story-illustrator-client/src/my_stories/my_stories.css

```diff
@@ -0,0 +1,17 @@
+#myStories {
```

Comment:
Repeat of earlier comment, let's get rid of CSS and use chakra ui styles wherever we can.